File: 3_24/3task.java
import java.util.Scanner;

public class Main {
    public static int EvenDigits(int number) {
        int count = 0;
        while (number > 0) {
            int digit = number % 10;
            if (digit % 2 == 0) {
                count++;
            }
            number /= 10;
        }
        return count;
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter the number of numbers: ");
        int n = scanner.nextInt();
        System.out.print("Enter the numbers separated by a space: ");
        for (int i = 0; i < n; ++i) {
            int number = scanner.nextInt();
            System.out.print(EvenDigits(number));
            if (i < n - 1) {
                System.out.print(" ");
            }
        }
        System.out.println();
    }
}

File: 3_24/3task.kt
import java.util.Scanner

fun evenDigits(number: Int): Int {
    var count = 0
    var num = number
    while (num > 0) {
        val digit = num % 10
        if (digit % 2 == 0) {
            count++
        }
        num /= 10
    }
    return count
}

fun main() {
    val scanner = Scanner(System.`in`)
    print("Enter the number of numbers: ")
    println()
    val n = scanner.nextInt()
    print("Enter the numbers separated by a space: ")
    println()
    for (i in 0 until n) {
        val number = scanner.nextInt()
        print(evenDigits(number))
        if (i < n - 1) {
            print(" ")
        }
    }
    println()
}